client/js/views/templates/Header.js

```diff
@@ -1,5 +1,5 @@
 module.exports = function( { model } ) {
     const navOptions = model.map(datum => { 
-    	return `<div data-js = 'navList'> ${ this.capitalizeWords(datum) } </div>` }).join('')
-    return `<nav> ${ navOptions } </nav>`
+    	return `<div data-js='navList'>${ this.capitalizeWords(datum) }</div>`}).join('')
+    return `<nav>${ navOptions }</nav>`
```

Comment:
General style:
```
module.exports = function( { model } ) {
     const navOptions = model.map(datum => { 
   	return `<div data-js='navList'>${ this.capitalizeWords(datum) }</div>`;
     }).join('')
     return `<nav>${ navOptions }</nav>`
```

Also, when using an arrow function with one statement, you don't need the `return`:

```
const navOptions = model.map(datum => `<div data-js='navList'>${ this.capitalizeWords(datum) }</div>` ).join('')
```